base_layer/core/src/lib.rs

```diff
@@ -31,20 +31,18 @@
 #[macro_use]
 extern crate bitflags;
 
-#[cfg(feature = "base_node")]
-pub mod blocks;
-#[cfg(feature = "base_node")]
-pub mod chain_storage;
-#[cfg(feature = "base_node")]
-pub mod consensus;
-#[cfg(feature = "base_node")]
-pub mod helpers;
-#[cfg(feature = "base_node")]
-pub mod mining;
-#[cfg(feature = "base_node")]
-pub mod proof_of_work;
-#[cfg(feature = "base_node")]
-pub mod validation;
+#[macro_use]
+mod macros;
+
+cfg_base_node! {
+    pub mod blocks;
+    pub mod chain_storage;
+    pub mod consensus;
+    pub mod helpers;
+    pub mod mining;
+    pub mod proof_of_work;
+    pub mod validation;
+}
```

Comment:
Does this break intellisense? 
It's the reason we moved away from a previous macro that did this

Reply:
It appears that is issue is fixed - no problems with intellisense :)

Reply:
Removed